contracts/deployables/freeze-voting/FreezeVotingBase.sol

```diff
@@ -135,6 +137,23 @@ abstract contract FreezeVotingBase is
         return $.isFrozen;
     }
 
+    /**
+     * @inheritdoc IFreezable
+     * @dev CRITICAL SECURITY FUNCTION: Returns the most recent freeze timestamp.
+     * This timestamp is NEVER cleared, even after unfreeze or auto-expiry.
+     * Used by guards to permanently invalidate all transactions timelocked before this time.
+     */
+    function lastKnownFreezeTime()
```

Comment:
Can we rename this to `lastFreezeTime()` ?

Reply:
good call, updated

Reply:
@tbwebb22 can you provide another review please?